### ISSUE TITLE
[MHV MR / Accessibility] Focused items obscured by expanded print/download menu

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#122282]
### 📝 User story
As a keyboard user, I want the item that currently has keyboard focus to be completely visible.

### 📝 Problem statement
Focused items are obscured when the print/download menu remains expanded, even after focus moves outside of the component.

### 📝 Details
<details><summary>Links, screenshots, steps to recreate</summary>

#### 🌌 Supporting screenshots, videos, code snippets

---

##### 🔗 From [Medical Records > Allergies and reactions](https://staging.va.gov/my-health/medical-records/allergies)

<img width="670" height="262" alt="Screenshot showing a partially visible focused element under an expanded print/download menu" src="https://github.com/user-attachments/assets/d7d36501-a941-4012-9e0f-25af0d662ac7" />

---

#### 👣 Steps to recreate
1. Navigate to the Medical Records > Allergies and reactions page.
2. Use the Tab key to navigate to the Print or download actions menu component; select Enter to expand the menu.
3. Use the Tab key to move focus through the menu items.
4. Use the Tab key to move focus to the next focusable element after the Print or download actions menu component
5. Notice that the focus indicator (yellow border) is outlining an element that is under the expanded menu component.
</details>

---

### 🔗 Point of contact
Sarah Horton (@sarahhortonadhoc)

### ⚠️ Severity level
- 2, Serious. Should be fixed in 1-2 sprints post-launch.

#### Accessibility Standards
- [Guideline 2.4 Navigable: Provide ways to help users navigate, find content, and determine where they are](https://www.w3.org/WAI/WCAG21/Understanding/navigable)
  - [SC 2.4.11 Focus Not Obscured (Minimum) (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum)

#### Experience Standards
- [Category Usability: User is able to easily know or learn how to use the interface to complete tasks](https://design.va.gov/about/experience-standards/#category-usability)
  - [User can identify an element and its state](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)

### 𝌣 Proposed solution or next steps
- Close the actions button menu component when focus moves out of the component.

### 🔎 Supporting resources
- [Focus management](https://design.va.gov/accessibility/focus-management)

### ✅ Acceptance criteria
- [x] The system closes the print/download actions menu button when focus moves outside of the component.
- [x] The user can see complete focusable elements and their complete focus indicator when they have focus.

### ✅ Possible tasks
- [x] Review, refine and acknowledge feedback.
- [x] Update ticket information (assignee, labels, project(s), etc.)
- [x] Fix and/or document decisions made.
- [x] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [x] Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

### 🔎 General resources

- USWDS Docs: [USWDS Documentation](https://designsystem.digital.gov/)
- VADS Docs: [VADS Experience Standards](https://design.va.gov/about/experience-standards/)
- VADS Docs: [VADS Components Documentation](https://design.va.gov/components/)
- VADS Docs: [VADS Components Storybook](https://design.va.gov/storybook/?path=/docs/about-introduction--docs)
- VA Platform Docs: [Accessibility defect severity rubric](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric)
- VA Github Docs: [List of common issue tickets](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/accessibility/common-issues) you can reference, copy, and add to.
- W3C Docs: [Web Content Accessibility Guidelines (WCAG) 2.2](https://www.w3.org/TR/WCAG22/)
- W3C Docs: [WCAG 2.2 Understanding Docs](https://www.w3.org/WAI/WCAG22/Understanding/)

**Comment by steveg-IAT** [comment]
@sarahhortonadhoc, this looks good to be handed off. 
Confirming that the issue was found while testing Medical Record, but the issue is global to the MHV system.

I'll jump in and work the solution for MedRecs... and verify that this can be applie to all the products.

**Comment by sarahhortonadhoc** [comment]
@steveg-IAT I added [additional annotations to the Figma file](https://www.figma.com/design/gGU9oX4QVqbYdxzXfTqTEj/Medical-Records---Milestone-2?node-id=22151-41579&t=kuwNaKDX6Y1WoqGK-1) to further define the focus and keyboard interactions with the component.

**Comment by steveg-IAT** [comment]
@sarahhortonadhoc , 

- added focus to the menu/toggle button on escape
- added focus check to existing escape unit test

will demo tomorrow (20251031) in our sync, before pushing.
